debezium-core/src/main/java/io/debezium/connector/common/BaseSourceTask.java

```diff
@@ -126,7 +123,7 @@ public final void start(Map<String, String> props) {
             // WorkerSourceTask calls us immediately after we return the empty list.
             // This turns into a throttling so we need to make a pause before we return
             // the control back.
-            PAUSE_WHEN_NOT_STARTED.pause();
+            Metronome.parker(Duration.of(2, ChronoUnit.SECONDS), Clock.SYSTEM).pause();
```

Comment:
That way we're creating a new metronome each time. Maybe it'd be better to initialize lazily if it's null originally?

Reply:
In that case we would need to clear it when the connector successfully starts otherwise the next restart loop would exhibit the same problem. As this is only expcetion handling and one per-two seconds object creation I opted for the simplest solution.

Reply:
Ok, makes sense.